check-license.sh

```diff
@@ -12,13 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#!/bin/bash
 
 exitcode=0
 for fname in `find . -name "*.py"`; 
 do
-  diff -q <(head -14 $fname) <(cat license-header.txt) > /dev/null
-  if [ $? -ne 0 ]
+  HEADER=$(head -1 $fname)
```

Comment:
lgtm, but... if you're fixing this, mind also making this a function? :) (by "this" I mean the guts that check the header and all that)

Reply:
I’d vote for replacing it with a python script…